### PR TITLE
Add set compose tree location function to compose full import

### PR DESCRIPTION
**Pull request by ycheng-aa**
JIRA: PDC-1356

### CODE REVIEW

**Comment by ycheng-aa** [comment]
I think the `scheme` should be `schema`. But compose tree location API it's `scheme`, just follow it to avoid confusion．

File: pdc/apps/compose/lib.py [hunk 1]
```diff
@@ -262,12 +264,52 @@ def compose__import_images(request, release_id, composeinfo, image_manifest):
     return compose_obj.compose_id, imported_images
 
 
+def _set_compose_tree_location(request, compose_id, composeinfo, location, url, scheme):
+    ci = productmd.composeinfo.ComposeInfo()
+    common_hacks.deserialize_wrapper(ci.deserialize, composeinfo)
+    num_set_locations = 0
+    synced_content = [item.name for item in ContentCategory.objects.all()]
```
Comment (simozhan) [inline pdc/apps/compose/lib.py]: It seems synced_content is set to ['debug', 'source', 'binary'].
Is it possible to get it from composeinfo or manifest file or get it from inputs?

File: pdc/apps/compose/views.py [hunk 2]
```diff
@@ -842,17 +848,21 @@ def create(self, request):
...
+                                                                                             data['release_id'],
+                                                                                             data['composeinfo'],
+                                                                                             data['rpm_manifest'],
+                                                                                             data['image_manifest'],
+                                                                                             data['location'],
+                                                                                             data['url'],
+                                                                                             data['scheme'])
         return Response(data={'compose': compose_id, 'imported rpms': imported_rpms,
-                              'imported images': imported_images},
+                              'imported images': imported_images, 'set_locations': imported_images},
```
Comment (simozhan) [inline pdc/apps/compose/views.py]: 'set_locations': imported_images ? or 'set_locations': set_locations

File: pdc/apps/compose/lib.py (hunk 1 above)
Comment (ycheng-aa) [inline pdc/apps/compose/lib.py]: This implementation is mentioned in JIRA. It looks can't get synced_content from inputs.

File: pdc/apps/compose/views.py (hunk 2 above)
Comment (ycheng-aa) [inline pdc/apps/compose/views.py]: Fixed in  b36bed5. Thank you very much for careful review.

**Comment by simozhan** [comment]
ok. That looks good to me. :+1:

**Comment by bliuredhat** [comment]
look good for me

**Pull request merged**